doc/source/additional/glossary.rst

```diff
@@ -227,9 +227,9 @@ Glossary
      A DBMS service that ensures data integrity when multiple transactions are concurrently executed in a multi-user environment. In a multi-threaded DWS
      environment, concurrency control ensures that database operations are safe and all database transactions remain consistent at any given time.
 
-   config
-     config is a special mongod that stores metadata of a DB instance. config is deployed as a replica set. To create a DB instance or modify metadata of a DB
-     instance, config must be available.
+   Config(DDS)
+     A `config` is a special Document Database Service process that stores metadata of a DB instance. A `config` is deployed as a replica set. To create
```

Comment:
In my opinion the sentence does not make sense

Reply:
I fixed that wording as well.